addons/terrabrush/Scripts/KeybindSettings.cs

```diff
@@ -0,0 +1,86 @@
+using Godot;
+using System.Text;
+
+namespace TerraBrush;
+
+[Tool]
+public partial class KeybindSettings : Godot.AcceptDialog
+{
+	[NodePath("VBoxContainer/Tree")]
+	private Tree _keybindList;
```

Comment:
```suggestion
	[NodePath("VBoxContainer/Tree")] private Tree _keybindList;
```

Same for the "unique name"